netsim/cli/connect.py

```diff
@@ -241,7 +242,7 @@ def run(cli_args: typing.List[str]) -> None:
   host = args.host
 
   if host in topology.nodes:
-    connect_to_node(args,rest,topology,log_level)
+    connect_to_node(args,rest,topology,host,log_level)
```

Comment:
Moving **host** to the first place (see above) would make this lovely similar to the next call ;)